recipes/netcdf-cxx4/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "4.2.1" %}
+
+package:
+  name: netcdf-cxx4
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/Unidata/netcdf-cxx4
+  git_tag: v{{ version }}
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
As soon as the feedstock is created let's open an issue to tackle the Windows build.

Ping @groutr who might have an idea about this.